apps/www/src/lib/registry/default/ui/select/select-item.svelte

```diff
@@ -18,7 +18,7 @@
 	{disabled}
 	{label}
 	class={cn(
-		"relative flex w-full cursor-default select-none items-center rounded-sm py-1.5 pl-8 pr-2 text-sm outline-none focus:bg-accent focus:text-accent-foreground data-[disabled]:pointer-events-none data-[disabled]:opacity-50",
+		"relative flex w-full cursor-default select-none items-center rounded-sm py-1.5 pl-8 pr-2 text-sm outline-none hover:bg-accent hover:text-accent-foreground data-[highlighted]:bg-accent data-[highlighted]:focus:text-accent-foreground data-[disabled]:pointer-events-none data-[disabled]:opacity-50",
```

Comment:
We shouldn't need `hover` classes here.

You also forgot to remove `focus` from one of the highlighted selectors 😃
should be - `data-[highlighted]:text-accent-foreground`